std/math/fp4_bb.asm

```diff
@@ -0,0 +1,203 @@
+use std::check::assert;
+use std::check::panic;
+use std::convert::fe;
+use std::convert::int;
+use std::convert::expr;
+use std::prover::eval;
+
+/// BETA = 11
+/// NBETA = 2013265910
+
+let BABYBEAR_PRIME = 0x78000001;
+
+/// An element of the extension field over the BabyBear Field
+/// relative to the irreducible polynomial X^4 + 11,
```

Comment:
I double-checked using Sage, and both $X^4 + 11$ and $X^4 - 11$ works for all the fields that we currently support.

I think we can keep the same instead of using $X^4 - 11$, cause there will be no efficiency difference, we can keep $X^4 = -11$ instead of $X^4 = 11$ since in the `inv_ext` we're doing a similar approach

Reply:
But, we have `2013265910`make the things not-field specific

Reply:
Okay, I'm going to try to make it field-specific rn